src/pagecache/io_uring.rs

```diff
@@ -186,6 +186,7 @@ impl<T> Uring<T> {
                     1,
                     off_t::try_from(offset).unwrap(),
                 );
+                io_uring_sqe_set_flags(sqe, IOSQE_FIXED_FILE);
```

Comment:
IOSQE_FIXED_FILE will use the index of registered fd list instead of fd itself.

Reply:
interesting, why does it even worked